InventoryInStorePickupSales/etc/extension_attributes.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Api/etc/extension_attributes.xsd">
+    <extension_attributes for="Magento\Sales\Api\Data\ShipmentInterface">
+        <attribute code="is_notified" type="boolean" />
```

Comment:
`is_notified` could be confusing out of In-Store Pickup scope.

What about `is_ready_for_pickup_notified`?